styles/cart.css

```diff
@@ -54,3 +58,29 @@ footer {
   top: 20rem;
   height: fit-content;
 }
+
+@media (max-width: 1038px) {
```

Comment:
prefer using relative units in media query

Reply:
ok will update all media queries at the end